app/services/licences/supplementary/create-licence-supplementary-year.service.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+/**
+ * Creates a licenceSupplementaryYears record based on the provided licence data
+ * @module CreateLicenceSupplementaryYearService
+ */
+
+const LicenceSupplementaryYearModel = require('../../../models/licence-supplementary-year.model.js')
+
+/**
+ * Creates a licenceSupplementaryYears record based on the provided licence data
+ *
+ * @param {module:LicenceModel} licenceId - The UUID of the licence to be persisted
+ * @param {Object[]} years - An array of the financial year ends to be persisted as individual records
```

Comment:
Ooh, really minor one!

```suggestion
 * @param {Object[]} financialYearEnds - An array of the financial year ends to be persisted as individual records
```